Apps/Src/ReadTritium.c

```diff
@@ -5,29 +5,86 @@
 #include "CANbus.h"
 #include <string.h>
 
-void Task_ReadTritium(void *p_arg)
-{
+//status limit flag masks
+#define MASK_MOTOR_TEMP_LIMIT (1<<6) //check if motor temperature is limiting the motor 6
 
+uint16_t Motor_FaultBitmap = T_NONE;
+
+
+/**
+ * @brief Return the static error field from this layer
+ * 
+ */
+tritium_error_code_t MotorController_getTritiumError(void){
+    //TODO: implement for loop to parse this
+	for(int i = 0; i < NUM_TRITIUM_ERRORS; ++i){
+		if(Motor_FaultBitmap & (1<<i)){
+			return ((tritium_error_code_t)(1<<i));
+		}
+	}
+}
```

Comment:
Also, is this used anywhere as of right now? I assume yall are planning on using this to differentiate fault behaviors inside the fault assertion thread, because if not, then it doesn't have many purposes.